src/nni_manager/config/kubeflow/tfjob-crd-v1.json

```diff
@@ -0,0 +1,17 @@
+{
+  "kind": "CustomResourceDefinition", 
+  "spec": {
+    "scope": "Namespaced", 
+    "version": "v1", 
+    "group": "kubeflow.org", 
+    "names": {
+      "kind": "TFJob", 
+      "plural": "tfjobs", 
+      "singular": "tfjob"
+    }
+  }, 
+  "apiVersion": "apiextensions.k8s.io/v1beta1",
```

Comment:
v1beta1?

Reply:
https://github.com/kubeflow/tf-operator/blob/master/examples/crd/crd-v1.yaml
They use it so here.